src/Common/ProfileEvents.cpp

```diff
@@ -341,6 +341,7 @@
     \
     M(WaitMarksLoadMicroseconds, "Time spent loading marks", ValueType::Microseconds) \
     M(BackgroundLoadingMarksTasks, "Number of background tasks for loading marks", ValueType::Number) \
+    M(SynchronousMarksTasks, "Number of times marks were loaded synchronously because they were already present in the cache.", ValueType::Number) \
```

Comment:
`SynchronousMarksTasks` is confusing, it is synchronous indeed, but only for cache hists,
`FastMarksTasks` how about that name?

`MarkCacheHits` is already present. But it is incremented in async tasks also.

Reply:
Agree that it is confusing. Let's use `MarksTasksFromCache`